standalone/src/models/product/product/product-variant/product-variant-draft/fields-config.ts

```diff
@@ -1,7 +1,10 @@
 import { fake, type TModelFieldsConfig } from '@/core';
-import { PriceDraft } from '@/models/commons';
-import { AttributeDraft } from '../../attribute';
-import { ImageDraft } from '../../image';
```

Comment:
Since I had to update the import, I also took the chance to use the new models instead of the legacy ones when available.